letsencrypt/crypto_util.py

```diff
@@ -45,8 +44,12 @@ def init_save_key(key_size, key_dir, keyname="key-letsencrypt.pem"):
         logger.exception(err)
         raise err
 
+    import zope.component
+    from letsencrypt import interfaces
```

Comment:
Safe to move these imports to the top, also reduces duplicate code in lines 79-80